draft-ietf-jose-fully-specified-algorithms.xml

```diff
@@ -661,7 +661,7 @@
 	</t>
       </section>
 
-      <section anchor="KEMs" title="KEMs and Encapsulated Keys">
+      <!-- <section anchor="KEMs" title="KEMs and Encapsulated Keys">
```

Comment:
Why is the KEMs section now commented out?

Reply:
A local editor mistake, as you can see in the latest changes, I've restructured the entire section to focus on the encryption modes that JOSE and COSE support, and explain what fully specified means for each mode.